### PR TITLE
[web] 030 permission

### PR DESCRIPTION
**Pull request by elliotmessi**
1. active user list
2. reg user list

### CODE REVIEW

File: ymir/web/src/config/routes.ts [hunk 1]
```diff
@@ -124,6 +124,30 @@ const Routes = [
         component: "@/pages/user/info",
         title: "keywords.title",
       },
+      {
+        path: "/home/mirror",
```
Comment (phoenix-xhuang) [inline ymir/web/src/config/routes.ts:128]: mirror是啥？镜像一般叫做docker image

File: ymir/web/src/locales/modules/common.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ const common = {
   'common.top.menu.configure': { cn: "系统配置", en: "Configure", },
   'common.top.menu.permission': { cn: "权限配置", en: "Permission", },
   'common.top.menu.resource': { cn: "资源配置", en: "Resource", },
+  'common.top.menu.mirror': { cn: "我的镜像", en: "Mirror", },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/common.ts]: Images

Comment (elliotmessi) [reply ymir/web/src/locales/modules/common.ts]: fixed

File: ymir/web/src/config/routes.ts (hunk 1 above)
Comment (elliotmessi) [reply ymir/web/src/config/routes.ts:128]: 后续会全部修正

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**